ksqldb-common/pom.xml

```diff
@@ -29,6 +29,12 @@
 
     <dependencies>
 
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <version>4.0.2</version>
```

Comment:
Add scope `test`

Reply:
I think the place where it's used is also getting called from non test code